typhos/display.py

```diff
@@ -1136,6 +1141,26 @@ def _get_specific_screens(cls, device_cls):
             if not utils.is_standard_template(template)
         ]
 
+    def to_image(self):
+        """
+        Return the entire display as a QtGui.QImage.
+
+        Returns
+        -------
+        QtGui.QImage
+            The display, as an image.
+        """
+        if self._display_widget is not None:
+            return utils.widget_to_image(self._display_widget)
+
+    @Slot()
+    def copy_to_clipboard(self):
+        """Copy the display image to the clipboard."""
+        image = self.to_image()
+        if image is not None:
+            clipboard = QtGui.QGuiApplication.clipboard()
+            clipboard.setImage(image)
```

Comment:
This isn't the point of the PR but it looks really useful

Reply:
:+1: Example is at the bottom of the PR description. Once that display got to be beyond 1 screen high, the tool became a necessity.